ideas/co-design-draft-proposal/index.html

```diff
@@ -0,0 +1,176 @@
+
+                <p>This toolkit would be used to discover, evaluate and provide reusable co-design activities that can be applied throughout the Sidewalk Toronto project; from the development of the MIDP to the building and subsequent habitation of the Quayside site. It aims to cut across all the work streams defined by Sidewalk Toronto to ensure the public is fully engaged throughout the process of designing Quayside. It will also include community-led measures to assess the effectiveness of the proposed principles and evolve them once the project has been built and is being lived in.</p>
+
+                <p>During the creation of the <em>Inclusion and Accessibility Principles</em>, we may find that some are in opposition to others. The toolkit will provide a framework in which to resolve those conflicts and provide clarity on why and how the principles have been created.</p>
+
+                <p>We think it is important to develop the <em>Inclusion and Accessibility Principles</em> concurrently with the inclusive design tools and the community-led measures due to the reasons mentioned here. Each component of this toolkit works with the other parts to support the <em>Inclusion and Accessibility Principles</em> and to provide a means that could be easily used by the community, and achieved by the Sidewalk team. If we focus on the creation of accessibility principles only, without supporting them with the design toolkit and the community-led measures, we risk not fully responding to the accessibility needs and requirements of the Quayside project.</p>
+
+                <p>Life in the Quayside neighbourhood and in Toronto will change; new technologies will be developed and our ideas of urban living will also change. The Toolkit will provide tools that we can use to continue to ensure that our city is inclusive, livable, and welcoming. Sidewalk Labs and Waterfront Toronto will also be able to reuse and adapt these tools for any other project to ensure the outcomes are more inclusive and accessible.</p>
+
+                <img src="proposal-diagram-02.png" alt="In this diagram, co-design is placed at the centre representing an engine that initiates an iterative cycle. This cycle includes three main components of perspectives, activities and reflection measures. The activities and reflection measures also include individual iterative cycles which contain implementation and application phases. This whole system empowered by co-design has two outputs: 'Inclusion and accessibility principles' and 'Inclusive design tools and civic metrics'." />
```

Comment:
This image renders pretty large for me. Let's check with Sepi if she wants it to be this big, and scale it otherwise.

Reply:
Also, this alt text is a bit on the long side but I think it's appropriate for the complexity of the image, so it's cool.